frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -123,12 +123,25 @@ final case class Project(
     }
   }
 
-  def pickValidResources: Array[AbsolutePath] = {
-    // Only add those resources that exist at the moment of creating the classpath
-    resources.iterator.filter(_.exists).toArray
+  def runtimeResources: List[AbsolutePath] = platform match {
+    case jvm: Platform.Jvm => jvm.resources
+    case _ => resources
   }
 
-  def fullClasspath(dag: Dag[Project], client: ClientInfo): Array[AbsolutePath] = {
+  def pickValidRuntimeResources(): Array[AbsolutePath] = platform match {
+    case jvm: Platform.Jvm => Project.pickValidResources(runtimeResources)
+    case _ => pickValidResources()
+  }
+
+  def pickValidResources(): Array[AbsolutePath] =
```

Comment:
The one in `Project` only does the filtering of the resources (check whether they exist). Should I rename it to `existingResources` or soemthing?

Reply:
I meant that this method no longer needs to exist here because we define it in the companion. It would be good if we redirect call-sites to this method to the companion object and remove this definition

Reply:
I understand now. I removed this method, old call sites now use `Project.pickValidResources`